docs/dev/table/connect.md

```diff
@@ -1075,6 +1075,94 @@ CREATE TABLE MyUserTable (
+
+For append-only queries, the connector can also operate in [append mode](#update-modes) for exchanging only INSERT messages with the external system.
+
+To use this connector, add the following dependency to your project:
+
+{% highlight xml %}
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-connector-hbase{{ site.scala_version_suffix }}</artifactId>
+  <version>{{ site.version }}</version>
+</dependency>
+{% endhighlight %}
+
+The connector can be defined as follows:
+
+<div class="codetabs" markdown="1">
+<div data-lang="YAML" markdown="1">
+{% highlight yaml %}
+connector:
+  type: hbase
+  version: 1.4.3                 # required: valid connector versions are "1.4.3"
+  
+  table-name: "hbase_table_name" # required: hbase table name
+  
+  zookeeper:
+    quorum: "localhost:2181"     # required: HBase Zookeeper quorum configuration
+    znode.parent: "/test"        # required: the root dir in Zookeeper for HBase cluster
+    
+  write.buffer-flush:
+    max-size: 1048576            # optional: Write option, sets when to flush a buffered request
+                                 # based on the memory size of rows currently added.
+    max-rows: 1                  # optional: Write option, sets when to flush buffered 
+                                 # request based on the number of rows currently added.
+    interval: 1                  # optional: Write option, sets a flush interval flushing buffered 
+                                 # requesting if the interval passes, in milliseconds.
+{% endhighlight %}
+</div>
+
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+CREATE TABLE MyUserTable (
+  hbase_rowkey_name rowkey_type,
+  hbase_column_family_name1 ROW<...>,
+  hbase_column_family_name2 ROW<...>
+) WITH (
+  'connector.type' = 'hbase', -- required: specify this table type is hbase
+  
+  'connector.version' = '1.4.3',          -- required: valid connector versions are "1.4.3"
+  
+  'connector.table-name' = 'hbase_table_name',  -- required: hbase table name
+  
+  'connector.zookeeper.quorum' = 'localhost:2181', -- required: HBase Zookeeper quorum configuration
+  'connector.zookeeper.znode.parent' = '/test',  -- required: the root dir in Zookeeper for HBase cluster
+
+  'connector.write.buffer-flush.max-size' = '1048576', -- optional: Write option, sets when to flush a buffered request
+                                                       -- based on the memory size of rows currently added.
+
+  'connector.write.buffer-flush.max-rows' = '1', -- optional: Write option, sets when to flush buffered 
+                                                    -- request based on the number of rows currently added.
+
+  'connector.write.buffer-flush.interval' = '1', -- optional: Write option, sets a flush interval flushing buffered 
```

Comment:
'connector.write.buffer-flush.interval' = '2s', -- optional: writing option, ............. Default value is "0s", which means no asynchronous flush thread will be scheduled.